viz-lib/src/visualizations/chart/Editor/XAxisSettings.jsx

```diff
@@ -14,6 +16,32 @@ export default function XAxisSettings({ options, onOptionsChange }) {
         onChange={xAxis => onOptionsChange({ xAxis })}
       />
 
+      {includes(["histogram"], options.globalSeriesType) && (
+        <React.Fragment>
+          <Section>
+            <InputNumber
+              label="Bin Size"
+              className="w-100"
+              placeholder="Auto"
+              data-test="Chart.XAxis.BinSize"
+              defaultValue={options.binSize}
+              onChange={binSize => onOptionsChange({ binSize: cleanNumber(binSize) })}
```

Comment:
@kravets-levko using toNumber() from AxisSettings resolves the non-updating issue mentioned earlier. Should I move the toNumber function as a common function and use it here?

Reply:
yes, sounds good

Reply:
Done in https://github.com/getredash/redash/pull/4426/commits/f1a8fe6e20df2a1b8210e6a225876ff83e3cb426